onnxruntime/test/testdata/transform/fusion/bias_gelu_add_inputs_swapped_gen.py

```diff
@@ -0,0 +1,41 @@
+import onnx
+from onnx import helper
+from onnx import TensorProto
+
+graph = helper.make_graph(
+    [  # nodes
+        # Identity nodes before Add in order to create graph edges to Add
+        helper.make_node("Identity", ["A"], ["A_out"], "identity_A"),
+        helper.make_node("Identity", ["B"], ["B_out"], "identity_B"),
+
+        # Add node before Gelu
+        helper.make_node("Add", ["B_out", "A_out"], ["add0_out"], "add0"),
+
+        # Gelu subgraph
+        helper.make_node("Div", ["add0_out", "div_const"], ["div_out"], "div"),
+        helper.make_node("Mul", ["add0_out", "mul_const"], [
+                         "mul_out"], "mul0"),
+        helper.make_node("Erf", ["div_out"], ["erf_out"], "erf"),
+        helper.make_node("Add", ["erf_out", "add_const"], [
+                         "add1_out"], "add1"),
+        helper.make_node("Mul", ["mul_out", "add1_out"], ["C"], "mul1"),
+    ],
+    "Gelu_Add_Fusion",  # name
+    [  # inputs
+        helper.make_tensor_value_info('A', TensorProto.FLOAT, [
+                                      'unk_1', 'unk_2', 3072]),
+        helper.make_tensor_value_info('B', TensorProto.FLOAT, [3072]),
+    ],
+    [  # outputs
+        helper.make_tensor_value_info('C', TensorProto.FLOAT, [
+                                      'unk_3', 'unk_4', 3072]),
```

Comment:
can we name the same 'unk_1' and 'unk_2' if the shape will be the identical to A?

Reply:
will revert the changes fixing BiasGelu fusion and use the changes from #4160